### PR TITLE
add logic to update TimeUnixNano from metrics

### PR DESCRIPTION
**Pull request by resdenia**

### CODE REVIEW

File: src/transformTs.js [hunk 1]
```diff
@@ -135,6 +135,41 @@ function checkNestedObjectByKeyValue(obj, objKey, objValue) {
     return false
 }
 
+function convertTimeUnixNano(metric){
```
Comment (yotamloe) [inline src/transformTs.js]: Why do we need to add this logic when you can just set `time.now()` for every sample? https://github.com/logzio/js-metrics/blob/main/src/transformTs.js#L4

Comment (resdenia) [reply src/transformTs.js]: @yotamloe  yeap! You're right! Thanks!

**Review by yotamloe — COMMENTED** [review]
Please change the recommended npm version: `npm install logzio-nodejs-metrics-sdk@0.2.1
` -> `npm install logzio-nodejs-metrics-sdk@0.3.0`

**Comment by yotamloe** [comment]
Also some of the tests are failing

**Review by yotamloe — APPROVED** [review]
Nicely done!

File: README.md [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
 
 Set the variables in the following code snippet:
 
-|Environment variable|Description|
-|---|---|
-|url|  The Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic. |
-|token| Your Logz.io Prometheus Metrics account token.  |
+| Environment variable | Description                                                                                                                         |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| url                  | The Logz.io Listener URL for for your region, configured to use port **8052** for http traffic, or port **8053** for https traffic. |
```
Comment (Doron-Bargo) [inline README.md]: Please add an example
Something like
....or port **8053** for https traffic. For example - https://listener.logz.io:8053 ...

File: package-lock.json [hunk 3]
```diff
@@ -1,13749 +1,5207 @@
...
-        "@grpc/proto-loader": "^0.6.0",
-        "@opentelemetry/api": "^1.0.0",
-        "@opentelemetry/core": "^1.0.1",
-        "@opentelemetry/exporter-metrics-otlp-http": "^0.27.0",
-        "@opentelemetry/exporter-trace-otlp-http": "^0.27.0",
-        "@opentelemetry/sdk-metrics-base": "^0.27.0",
-        "google-protobuf": "^3.17.3",
-        "nock": "^13.1.1",
-        "protobufjs": "^6.9.0",
-        "requestretry": "^5.0.0",
```
Comment (Doron-Bargo) [inline package-lock.json:21]: Please update relevant packages according to Snyk issues

**Review by Doron-Bargo — COMMENTED** [review]
looking good
Please update the relevant dependencies according to Snyk report

**Pull request merged**